db-server/index.js

```diff
@@ -239,6 +239,22 @@ function createServer(db) {
     op((req) => db.consumeRecoveryCode(req.params.id, req.params.code))
   )
 
+  api.get('/account/:id/recoveryKeys/:recoveryKeyId',
+    op((req) => {
+      return db.getRecoveryKey(req.params.id, req.params.recoveryKeyId)
+    })
+  )
+
+  api.del('/account/:id/recoveryKeys/:recoveryKeyId',
+    op((req) => {
+      return db.deleteRecoveryKey(req.params.id, req.params.recoveryKeyId)
+    })
+  )
+
+  api.post('/account/:id/recoveryKeys',
+    op((req) => db.createRecoveryKey(req.params.id, req.body))
```

Comment:
We seem to have got out of the habit of using the helpers for these to reduce boilerplate. This one can be implemented using `withIdAndBody` right? And can the two above use `withParams`?